traits/tests/test_ctraits.py

```diff
@@ -244,3 +245,33 @@ def get_handler_refcount():
         tr.handler = None
         self.assertFalse(finalizer.alive)
         self.assertIsNone(tr.handler)
+
+    def test_invalid_trait_initialization(self):
+        self.assertNotIn(9, TraitKind)
```

Comment:
This line is failing in Python 3.8:

```
TypeError: unsupported operand type(s) for 'in': 'int' and 'EnumMeta'
```

I assume your intent was to make sure that you were using a value that wasn't in `TraitKind`, which makes sense. One possibility for fixing this would be to use `max(TraitKind) + 1` in place of `9`.

Reply:
Yes that was the intent and I will replace it with the suggested fix.